config/routes.rb

```diff
@@ -72,4 +17,10 @@
   get 'profile/:id/edit' => 'profile#edit', as: :edit_profile
   put 'profile/:id' => 'profile#update', as: :profile
   delete 'profile/:id' => 'profile#destroy'
+
+  resources :blocks
+  post 'blocks/:id/set_as_current' => 'blocks#set_as_current',
+       as: :block_set_as_current
+  post 'blocks/:id/reset_as_current' => 'blocks#reset_as_current',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.